lib/open_food_network/order_cycle_management_report.rb

```diff
@@ -33,44 +48,61 @@ def table_items
     end
 
     def filter(search_result)
-      filter_to_payment_method filter_to_shipping_method filter_to_order_cycle search_result
+      filtered_orders = filter_to_order_cycle(search_result)
+      filtered_orders = filter_to_shipping_method(filtered_orders)
+      filter_to_payment_method(filtered_orders)
     end
 
     private
 
     def payment_method_row(order)
-      ba = order.billing_address
-      [ba.firstname,
-       ba.lastname,
-       order.distributor.andand.name,
-       customer_code(order.email),
-       order.email,
-       ba.phone,
-       order.shipping_method.andand.name,
-       order.payments.first.andand.payment_method.andand.name,
-       order.payments.first.amount,
-       OpenFoodNetwork::UserBalanceCalculator.new(order.email, order.distributor).balance
+      billing_address = order.billing_address
+      [
+        billing_address.firstname,
+        billing_address.lastname,
+        order.distributor.andand.name,
+        customer_code(order.email),
+        order.email,
+        billing_address.phone,
+        shipping_method_to_display(order).name,
+        order.payments.first.andand.payment_method.andand.name,
+        order.payments.first.amount,
+        OpenFoodNetwork::UserBalanceCalculator.new(order.email, order.distributor).balance
       ]
     end
 
     def delivery_row(order)
-      sa = order.shipping_address
-      [sa.firstname,
-       sa.lastname,
-       order.distributor.andand.name,
-       customer_code(order.email),
-       "#{sa.address1} #{sa.address2} #{sa.city}",
-       sa.zipcode,
-       sa.phone,
-       order.shipping_method.andand.name,
-       order.payments.first.andand.payment_method.andand.name,
-       order.payments.first.amount,
-       OpenFoodNetwork::UserBalanceCalculator.new(order.email, order.distributor).balance,
-       has_temperature_controlled_items?(order),
-       order.special_instructions
+      shipping_address = order.shipping_address
+      [
+        shipping_address.firstname,
+        shipping_address.lastname,
+        order.distributor.andand.name,
+        customer_code(order.email),
+        "#{shipping_address.address1} #{shipping_address.address2} #{shipping_address.city}",
+        shipping_address.zipcode,
+        shipping_address.phone,
+        shipping_method_to_display(order).name,
+        order.payments.first.andand.payment_method.andand.name,
+        order.payments.first.amount,
+        OpenFoodNetwork::UserBalanceCalculator.new(order.email, order.distributor).balance,
+        has_temperature_controlled_items?(order),
+        order.special_instructions
       ]
     end
 
+    # Returns the appropriate shipping method to display for the given order
+    #
+    # @param order [Spree::Order]
+    # @return [#name]
+    def shipping_method_to_display(order)
+      shipment = order.shipments.last
```

Comment:
In places like these (reports) some thought on the UI is required. An order with multiple shipments won't be displayed in a table now.